incubator/vault/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Vault, a tool for managing secrets
 name: vault
-version: 0.14.7
+version: 0.15.7
```

Comment:
This should be 0.15.0

Reply:
Hey Towmeykaw. I've just pushed the commit that sets the version number to 0.15.0. 
Is there anything else?

Reply:
Looks fine to me. But I'm not a maintainer